crates/router/src/core/payment_methods/cards.rs

```diff
@@ -1978,6 +1978,31 @@ pub async fn delete_card_from_locker(
     .await
 }
 
+#[cfg(all(feature = "v2", feature = "customer_v2"))]
+pub async fn delete_card_from_locker_by_global_id(
```

Comment:
this should be by locker_id and not global_id

Reply:
```
pub async fn delete_card_from_locker(
    state: &routes::SessionState,
    customer_id: &id_type::CustomerId,
    merchant_id: &id_type::MerchantId,
    card_reference: &str,
) -> errors::RouterResult<payment_methods::DeleteCardResp> {
    metrics::DELETE_FROM_LOCKER.add(&metrics::CONTEXT, 1, &[]);

    common_utils::metrics::utils::record_operation_time(
        async move {
            delete_card_from_hs_locker(state, customer_id, merchant_id, card_reference)
                .await
                .inspect_err(|error| {
                    metrics::CARD_LOCKER_FAILURES.add(&metrics::CONTEXT, 1, &[]);
                    logger::error!(?error, "Failed to delete card from locker");
                })
        },
        &metrics::CARD_DELETE_TIME,
        &metrics::CONTEXT,
        &[],
    )
    .await
}
```

but the v1 function of it doesnot delete it by locker_id! basically i think it's an api call to locker where we get locker info from the state

Reply:
This function itself not relevant to v2 just make it todo or deprecate it.

Reply:
okay!